src/main/java/uk/gov/hmcts/reform/prl/services/ServiceOfApplicationPostService.java

```diff
@@ -219,33 +224,32 @@ public List<Document> getStaticDocs(String auth, CaseData caseData) {
 
     private BulkPrintDetails sendBulkPrint(CaseData caseData, String authorisation,
                                           List<Document> docs, Address address, String name, String servedParty) {
-        List<Document> sentDocs = new ArrayList<>();
         ZonedDateTime zonedDateTime = ZonedDateTime.now(ZoneId.of("Europe/London"));
         String currentDate = DateTimeFormatter.ofPattern("dd MMM yyyy HH:mm:ss").format(zonedDateTime);
         String bulkPrintedId = "";
         try {
             log.info("*** Initiating request to Bulk print service ***");
             log.info("*** number of files in the pack *** {}", null != docs ? docs.size() : "empty");
-            UUID bulkPrintId = bulkPrintService.send(
-                String.valueOf(caseData.getId()),
-                authorisation,
-                LETTER_TYPE,
-                docs
-            );
-            log.info("ID in the queue from bulk print service : {}", bulkPrintId);
-            bulkPrintedId = String.valueOf(bulkPrintId);
-            sentDocs.addAll(docs);
-
+            if (launchDarklyClient.isFeatureEnabled("soa-bulk-print")) {
```

Comment:
@shashisk97 do we need this flag anymore?

Reply:
@shashisk97 please confirm.
I can remove if not needed anymore.